src/App.jsx

```diff
@@ -1,20 +1,53 @@
-import React from "react";
+import React, { useState } from "react";
 
 //IMPORT STYLES
 import { GlobalStyles } from "./App.styles";
 
-//TODO: IMPORT COMPONENTS
+//IMPORT COMPONENTS
 import Footer from "@molecules/footer/Footer";
 import Header from "@molecules/header/Header";
+import When from "@atoms/when/When";
+import Where from "@atoms/where/Where";
+import How from "@atoms/how/How";
 
 //TODO: IMPORT RESET CSS
 
 const App = () => {
+  //HOOKS
+  const [isConfirmed, setConfirmed] = useState(false);
+  const [isModalOpen, setModalOpen] = useState(false);
+
+  //HANDLE FUNCTIONS
+  const handleConfirmed = () => {
+    console.log("Link clicked. Triggering Confirm button.");
+    return setConfirmed(!isConfirmed);
+  };
+  const handleModalOpen = () => {
+    setModalOpen(true);
+  };
+
+  const handleModalClose = () => {
+    setModalOpen(false);
+  };
+
   return (
     <>
       <GlobalStyles />
-      <Header />
-      <div style={{ margin: "35rem 0" }}>CONTENT TO DO</div>
+      <Header
+        onConfirmClick={handleConfirmed}
+        isModalOpen={isModalOpen}
+        onModalOpen={handleModalOpen}
+        onModalClose={handleModalClose}
+      />
+      <section id="when" style={{ margin: "35rem 0" }}>
```

Comment:
- [ ] remove all the section `html` from here. And make sure that each styled component you import **IS** a section
- [ ] make sure to don't forgot the ID
- [ ] Remove the hardcoded styles